src/main/kotlin/com/cognifide/gradle/aem/instance/tasks/Backup.kt

```diff
@@ -22,11 +27,28 @@ open class Backup : ZipTask() {
         entryCompression = ZipEntryCompression.STORED
     }
 
+    @Input
+    var uploadUrl = aem.props.string("aem.backup.uploadUrl")
+
+    @TaskAction
+    override fun copy() {
+        super.copy()
+        upload()
+    }
+
+    private fun upload() {
+        uploadUrl?.let { url ->
+            val backupZip = archiveFile.get().asFile
+            logger.lifecycle("Uploading backup: ${backupZip.path} to $url")
+            fileTransfer(url).upload(backupZip)
```

Comment:
finally it will be `aem.fileTransfer(url).upload(backupZip)` or `aem.fileTransfer.upload(url, backupZip)` and I am voting for option no 2 :)